dashboard/app/models/script.rb

```diff
@@ -1428,6 +1430,12 @@ def finish_url
     nil
   end
 
+  # A script that the general public can assign. Has been soft or
+  # hard launched.
+  def launched?
```

Comment:
Not to start too big of a discussion here, but why launched over assignable?

Reply:
I actually started with this as assignable and then changed it to launched. Assignable is already used in our system and is kinda complicated in that something like a pilot course can be assignable for certain people so I felt like launched was more clear.

Reply:
oh good point about pilots, I like launched better now as well :)